libs/elasticsearch/README.md

```diff
@@ -110,3 +110,71 @@ chat_history = ElasticsearchChatMessageHistory(
     es_api_key="your-api-key",
 )
 ```
+
+
+### ElasticsearchCache
+
+A caching layer for LLMs that exploits Elasticsearch.
+
+Simple example:
+
+```python
+from langchain.globals import set_llm_cache
+from langchain_elasticsearch.cache import ElasticsearchCache
```

Comment:
Could you export this class in `libs/elasticsearch/__init__.py` and change the import to
```py
from langchain_elasticsearchimport ElasticsearchCache
```

Reply:
done

Reply:
It looks like the import mentions in the README still use the `.cache` import.